easybuild/easyconfigs/l/LuaJIT2/LuaJIT2-v2.1-20220411-GCC-9.3.0.eb

```
@@ -0,0 +1,31 @@
+# This is the LuaJIT fork from openresty, not to be mixed up
+# with the original LuaJIT version!
+# Autor: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'ConfigureMake'
+
+name = "LuaJIT2"
+version = "v2.1-20220411"
+
+homepage = "https://github.com/openresty/luajit2"
+description = """openresty/luajit2 - OpenResty's maintained branch of LuaJIT.
+LuaJIT is a Just-In-Time Compiler (JIT) for the Lua
+programming language. Lua is a powerful, dynamic and light-weight programming
+language. It may be embedded or used as a general-purpose, stand-alone
+language. """
+
+toolchain = {'name': 'GCC', 'version': '9.3.0'}
+
+source_urls = ['https://github.com/openresty/luajit2/archive']
+sources = ['%(version)s.tar.gz']
+checksums = ['d3f2c870f8f88477b01726b32accab30f6e5d57ae59c5ec87374ff73d0794316']
+
+skipsteps = ['configure']
+installopts = 'PREFIX=%(installdir)s'
+
+sanity_check_paths = {
+    'files': ["bin/luajit"],
+    'dirs': []
+}
+
```

Let's also add a sanity check command:

```suggestion

sanity_check_commands = ['luajit -v']

```

---

I must have overlooked that. Thanks.